### PR TITLE
Programmatic prompts

### PR DESCRIPTION
**Pull request by GoBenGoh**

### CODE REVIEW

File: LLM Testing/Code-Testing/src/main/java/ChatGPTAPI.java [hunk 1]
```diff
@@ -51,7 +55,144 @@ public static void prompts() {
...
+        Gson gson = new GsonBuilder().setPrettyPrinting().disableHtmlEscaping().create();
+        JsonElement jsonElement = gson.toJsonTree(newRequest);
+        String jsonString = gson.toJson(jsonElement);
+        try(PrintWriter out = new PrintWriter(request)){
+            out.println(jsonString);
+        }
+        try(PrintWriter out = new PrintWriter("src/main/resources/newPrompt.txt")){
+            out.println(newPrompt);
+        }
+        //sendRequest(request, args[0]);
```
Comment (cmrnnathan) [inline LLM Testing/Code-Testing/src/main/java/ChatGPTAPI.java:164]: Should this be uncommented?

**Review by cmrnnathan — COMMENTED** [review]
Looks good. Just had a question about a commented line.

File: LLM Testing/Code-Testing/src/main/java/ChatGPTAPI.java (hunk 1 above)
Comment (GoBenGoh) [reply LLM Testing/Code-Testing/src/main/java/ChatGPTAPI.java:164]: Leave it uncommented so that we dont unintentionally call GPT 10 times in a row

**Review by cmrnnathan — APPROVED** [review]
Looks good.

**Pull request merged**